src/libs/Navigation/linkingConfig/CENTRAL_PANE_TO_RHP_MAPPING.ts

```diff
@@ -37,6 +37,8 @@ const CENTRAL_PANE_TO_RHP_MAPPING: Partial<Record<CentralPaneName, string[]>> =
         SCREENS.SETTINGS.WALLET.CARD_ACTIVATE,
         SCREENS.SETTINGS.WALLET.REPORT_VIRTUAL_CARD_FRAUD,
         SCREENS.SETTINGS.WALLET.CARDS_DIGITAL_DETAILS_UPDATE_ADDRESS,
+        SCREENS.SETTINGS.WALLET.VERIFY_ACCOUNT,
+        SCREENS.SETTINGS.ADD_BANK_ACCOUNT,
```

Comment:
Coming from https://github.com/Expensify/App/issues/50214 this caused an issue for `Add bank account` buttons that shouldn't show the settings page as the central pane.